ergo/metaculus.py

```diff
@@ -192,28 +201,81 @@ def sample_community(self):
         raise NotImplementedError("This should be implemented by a subclass")
 
     @staticmethod
-    def to_dataframe(questions: List["MetaculusQuestion"]) -> pd.DataFrame:
+    def to_dataframe(
+        questions: List["MetaculusQuestion"], extra_columns: List[str] = []
```

Comment:
I think the API would be clearer if `to_dataframe` took an optional `columns` keyword argument that defaulted to an initial value like `["id", "name", "title", "resolve_time"]` but allowed users to overwrite it. This way, users can control all columns that appear and the order they appear in, and the calling code makes it obvious which columns will be part of the result.